example/user/user.override.go

```diff
@@ -1,14 +0,0 @@
-package user
```

Comment:
Probably not intended to be deleting this file, either

Reply:
thanks, got it